terraform/aws/implementation/modules/s3/iam.tf

```diff
@@ -0,0 +1,54 @@
+# TODO iam permissions to speak with clear
+resource "aws_s3_bucket_policy" "allow_s3_access_for_ecr_viewer" {
+  bucket = aws_s3_bucket.s3.id
+  policy = data.aws_iam_policy_document.ecr_viewer_s3_policy.json
+}
+
+data "aws_iam_policy_document" "ecr_viewer_s3_policy" {
```

Comment:
I've been trying to follow a convention of putting all `data` blocks in the `data.tf` file of each module - would you mind moving this and other data blocks over as well?

Reply:
Done!